schema.yml

```diff
@@ -599,8 +807,8 @@ components:
         implicit:
           authorizationUrl: https://test.openpodcastapi.com/oauth/authorize
           scopes:
-            write:subscriptions: modify subscription information for your account
-            read:subscriptions: read your subscription information
+            write:subscriptions: modify subscription information & related episodes for your account
```

Comment:
I think reading & writing subscriptions & episodes can both be in the same scope. Would you agree @Sporiff @JonOfUs?